<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAP-1547" ignored="0" name="Test" tests="1" failures="1" timestamp="02 Feb 2020 15:14:28 GMT" time="47.151" errors="0">
  <testcase name="verifyTelecelRecharge" time="46.269" classname="com.Test.testCases.CreateRechargeTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Recarga   Recarga Saldo   de Telcel   al número 5523261151   de   $10 pesos] but found [Recarga   Recarga Saldo  de Telcel  al número 5523261151  de   $10 pesos]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Recarga   Recarga Saldo   de Telcel   al número 5523261151   de   $10 pesos] but found [Recarga   Recarga Saldo  de Telcel  al número 5523261151  de   $10 pesos]
at com.Test.testCases.CreateRechargeTest.verifyTelecelRecharge(CreateRechargeTest.java:47)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- verifyTelecelRecharge -->
</testsuite> <!-- Test -->
